src/core/plugins/Allergies/AllergiesEdit.js

```diff
@@ -8,16 +8,29 @@ import {
     LongTextInput
 } from "react-admin";
 
-const AllergiesEdit = props => (
-    <Edit title="Edit Allergy" redirect="show" {...props}>
-        <SimpleForm>
-            <TextInput source="cause" label="Cause" />
-            <LongTextInput source="reaction" label="Reaction / Description" />
-            <DisabledInput source="source" label="Source" />
-            <DisabledInput source="author" label="Author" />
-            <DisabledInput source="date" label="Date" />
-        </SimpleForm>
-    </Edit>
-);
+import EditToolbar from "../../common/EditToolbar";
+
+/**
+ * This component returns block with edit form for Allergies
+ *
+ * @author Bogdan Shcherban <bsc@piogroup.net>
+ * @param {shape} props
+ * @constructor
+ */
+const AllergiesEdit = props => {
+    const { classes } = props;
```

Comment:
I advise you to use the following syntax instead:

```jsx
cont AllergiesEdit = ({ classes, ...rest }) => (
        <Edit className={classes.edit} title="Edit Allergy" {...rest}>
            <SimpleForm toolbar={<EditToolbar />}>
                <TextInput source="cause" label="Cause" />
                <LongTextInput source="reaction" label="Reaction / Description" />
                <DisabledInput source="source" label="Source" />
                <DisabledInput source="author" label="Author" />
                <DisabledInput source="date" label="Date" />
            </SimpleForm>
        </Edit>
)
```

It's more concise and readable IMO

Reply:
thanks @fzaninotto 

@BogdanScherban

Reply:
@fzaninotto @tony-shannon 

I changed syntax according to example and updated PR

Reply:
@fzaninotto are you happy for me to merge this PR now then/would you like to do so?

Reply:
please do!